### PR TITLE
Engineer Gaming

### PR DESCRIPTION
**Pull request by alsoandanswer**
- imageadd: "Adds new hardhat, firesuit and firesuit helmet sprites. Ported and modified from /tg/ and Bee."
  - tweak: "Red firefighting helmets have been superceded by the Atmos firesuit helmets."
  - rscdel : "Golems no longer can wear firesuits. They literally can't wear anything else anyway."
  - rscadd: "Zavodskoi and Hephaestus Engineers now spawn with hardhats which match their uniforms."
  - rscadd: "New green hardhat! Enjoy Hephaestus green! All other hardhats tweaked to match uniforms!"
  - imageadd: "Resprites EMT helmet."
dreamseeker_PrJek2UBte

### CODE REVIEW

File: code/game/jobs/faction/hephaestus.dm [hunk 1]
```diff
@@ -69,18 +69,21 @@
 	name = "Engineer - Hephaestus"
 
 	uniform = /obj/item/clothing/under/rank/engineer/heph
+	head = /obj/item/clothing/head/hardhat/red
```
Comment (NonQueueingMatt) [inline code/game/jobs/faction/hephaestus.dm:72]: No. Same issue as said before, head items fuck over loadout preview.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/jobs/faction/hephaestus.dm (hunk 1 above)
Comment (alsoandanswer) [reply code/game/jobs/faction/hephaestus.dm:72]: the default engineer parent spawns with a hardhat as well

**Comment by alsoandanswer** [comment]
!review

**Comment by TheStryker** [comment]
I like the new hardhats but the firesuits seem too much like voidsuits (especially with the legs and how bulky they are) and they don't really seem like they match the current style we have on server.

**Comment by alsoandanswer** [comment]
> I like the new hardhats but the firesuits seem too much like voidsuits (especially with the legs and how bulky they are) and they don't really seem like they match the current style we have on server.

Well, really the bulkiness is in the same vein as the emergency softsuits. Coupled by the fact that they're able to be worn by a Vaurca Bulwark as well it isn't too far fetched

As for the style I am trying to rid ourselves of very old 2012 /tg/ style sprites by updating them with more contemporary versions

**Comment by SleepyGemmy** [comment]
> I like the new hardhats but the firesuits seem too much like voidsuits (especially with the legs and how bulky they are) and they don't really seem like they match the current style we have on server.

the regular firesuit is fine as it's a full-body covering item designed to protect you from heat and flames.
the atmos firesuit could honestly be the new atmos voidsuit sprite and the atmos firesuit can probably be scrapped, to be honest.

**Comment by Forester40** [comment]
> Well, really the bulkiness is in the same vein as the emergency softsuits. Coupled by the fact that they're able to be worn by a Vaurca Bulwark as well it isn't too far fetched

But, bulwarks can wear almost everything. I thought we were just supposed to accept that clothes became larger when a bulwark puts them on.

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**